src/syft/core/message/syft_message.py

```diff
@@ -1,7 +1,17 @@
 from ...common.message import AbstractMessage
-
+from ...common.id import UID
+from ...io import Route
 
 class SyftMessage(AbstractMessage):
-    def __init__(self, route, msg_id=None):
+    def __init__(self, route: Route, msg_id: UID = None) -> None:
         self.route = route
         self.msg_id = msg_id
+
+class SignedMessage(SyftMessage):
```

Comment:
Very nice concept - seems like the right place for this kind of class to live as well.